tools/fileconv/api3/parameters.go

```diff
@@ -21,6 +22,16 @@ func DataObjectCheck(objectName, fieldName string) bool {
 	return fieldName == "data"
 }
 
+// CustomMappingObjectCheck builds ObjectCheck using mapping,
+// which knows exceptions and patterns to determine response field name.
+func CustomMappingObjectCheck(dict *handy.DefaultMap[string, string]) ObjectCheck {
```

Comment:
Can you add an example as a comment? This is unclear by itself

Reply:
New method documentation header:
https://github.com/amp-labs/connectors/blob/13184db44673c35ef70bc145c42ee70c86014272/tools/fileconv/api3/parameters.go#L25-L38